tgui/packages/tgui/interfaces/OreRedemption.js

```diff
@@ -27,7 +28,8 @@ export const OreRedemption = (properties, context) => {
             <IdDisk height="100%" />
           </Flex.Item>
           <Flex.Item grow="1" overflow="hidden">
-            <Ores height="100%" />
+            <Sheet height="43%" />
```

Comment:
I think this may be causing all that empty space in your sections

Reply:
I know that this is the one causing the empty space in "Sheets" problem is that, if I lower it, the. The alloys become bigger, and also any new additions to sheets tab also requires scrolling. So I left it like that, not sure how I can make it so it expands the menu itself right now, if that is possible that is

Reply:
think I got something for this. gotta test later however